api-gateway/src/main/resources/static/scripts/Inventory-form/inventory-form.controller.js

```diff
@@ -5,7 +5,12 @@ angular.module('inventoryForm')
         var self = this;
         console.log("State params: " + $stateParams)
         $scope.inventoryTypeFormSearch = "";
-        $scope.inventoryTypeOptions = ["New Type", "Internal", "Sales"] //get all form the inventory type repository but dont remove the New Type
+        $scope.inventoryTypeOptions = ["New Type"] //get all form the inventory type repository but dont remove the New Type
```

Comment:
Mind explaining this code and why you removed Internal and Sales?

Reply:
No problem, since I am getting all inventory types, there is no need to hard code Internal and Sales inside the array, since it is all supposed to be fetched with the get all inv types request. We keep "New Type" since it is tied to the add inventory type request functionality.

Reply:
I appreciate the response. Good job on the task.